test/northstar-client.test.js

```diff
@@ -105,39 +106,26 @@ describe('NorthstarClient', () => {
       user.should.be.an.instanceof(NorthstarUser);
       user.should.be.an.instanceof(NorthstarUserAuthorized);
 
-      // Ensure properties and test values.
-      user.should.have.properties({
-        isAuthorized: true,
-        id: '5480c950bffebc651c8b456f',
-        firstName: 'test',
-        lastName: 'Last',
-        lastInitial: 'L',
-        photo: 'https://avatar.dosomething.org/uploads/avatars/5480c950bffebc651c8b456f.jpeg',
-        email: 'test@dosomething.org',
-        mobile: '5555555555',
-        interests: [
-          'interest number 1',
-          'int num 2',
-        ],
-        birthdate: '1989-05-04 00:00:00',
-        addrStreet1: '123',
-        addrStreet2: '456',
-        addrCity: 'Paris',
-        addrState: 'Florida',
-        addrZip: '555555',
-        source: 'phoenix',
-        mobilecommonsID: null,
-        parseInstallationIds: [
-          'parse-test',
-        ],
-        mobilecommonsStatus: null,
-        language: 'en-global',
-        country: 'US',
-        drupalID: '187',
-      });
+      const privateUserProperties = [
```

Comment:
I liked how you moved public properties to the top. Let's move this one as well!